templates/components/file.build.hbs

```diff
@@ -2,8 +2,8 @@
   <i class="fa fa-plus" aria-hidden="true"></i>
   <span v-if="selectedFileName">Replace file</span>
   <span v-else>Choose file</span>
-  <input type="file" ref="fileInput" :id="name" :name="name" :data-folder-id="mediaFolderId" class="input-file selectfile" :required="required" v-on:change="updateValue()">
+  <input type="file" ref="fileInput" :id="name" :name="name" :data-folder-id="mediaFolderId" class="input-file selectfile" :required="required" v-on:change="updateValue()" multiple>
```

Comment:
I also find that when the field is "required", validation triggers an error because the `[type="file"]` field is hidden with `display: none`, which makes the field **unfocusable**. Is it easy to fix this too?

Reply:
@tonytlwu fixed https://github.com/Fliplet/fliplet-widget-form-builder/pull/122/commits/a71bb85c5fc1e854d9450f4dd290c2cf927623ed